utils/s2n_random.c

```diff
@@ -325,24 +325,43 @@ int s2n_get_rdrand_data(struct s2n_blob *out)
     struct s2n_stuffer stuffer = {{0}};
     union {
         uint64_t u64;
+        struct {
+            /* since we check first that we're on intel, we can safely assume little endian. */
+            uint32_t uLow;
+            uint32_t uHigh;
```

Comment:
This would be our just upper case letters in a variable name. I mean that's not wrong, but would u_low, or just "low" be ok?

Reply:
I think u_low is more consistent with the rest of the code-base, but I was trying to be consistent with the rest of the struct.

Reply:
I can do u_low if you'd prefer